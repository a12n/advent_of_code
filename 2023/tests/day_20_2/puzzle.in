../day_20_1/puzzle.in