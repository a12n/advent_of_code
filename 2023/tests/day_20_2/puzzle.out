243081086866483
